src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -62,17 +62,17 @@ def test_csum_error_logging(self):
         host_devices = get_dmg_smd_info(dmg.storage_query_list_devices, 'devices')
         for host, devices in host_devices.items():
             for device in devices:
-                for entry in ('uuid', 'tgt_ids', 'role_bits', 'roles'):
+                for entry in ('uuid', 'tgt_ids', 'role_bits'):
                     if entry not in device:
                         self.fail(
                             'Missing {} info from dmg storage query list devices'.format(entry))
                 self.log.info(
-                    'Host %s device: uuid=%s, targets=%s, role=%s, role_bits=%s',
-                    host, device['uuid'], device['tgt_ids'], device['roles'], device['role_bits'])
+                    'Host %s device: uuid=%s, targets=%s, role_bits=%s',
+                    host, device['uuid'], device['tgt_ids'], device['role_bits'])
                 if not device['tgt_ids']:
                     self.log_step('Skipping device without targets on {}'.format(device['uuid']))
                     continue
-                if device['roles'] and not int(device['role_bits']) & 1:
+                if not int(device['role_bits']) & 1:
```

Comment:
I don't think this will work for MD-on-PMem mode where 'role_bits' will be 0. Either use approach in https://github.com/daos-stack/daos/pull/13667 or 
```suggestion
                if int(device['role_bits']) > 0 and not (int(device['role_bits']) & 1):
```

Reply:
It looks like the CI run didn't execute the desired test: https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-13660/2/pipeline/